controller/consul.go

```diff
@@ -52,7 +53,11 @@ func NewConsulMon(addr string) (*ConsulMon, error) {
 
 func (c *ConsulMon) queryServices() ([]*App, error) {
 	var apps []*App
-	addr := c.addr + fmt.Sprintf("%s/%s", nodeURL, c.node)
+	stale := ""
```

Comment:
This is fine but I think that `var s string` would have been slightly more clear for this declaration.